Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/Math/Filter.azsli

```diff
@@ -42,96 +42,10 @@ bool IsInsideOfImageSize(
         IsInsideOfImageSize(coord, outputImageSize);
 }
 
-//! This returns filtered value of "source" with weights in "filterTable" in 1 direction.
-//! @param coord the center coordinate (in Texture2DArray) of the filtered area.
-//!              the xy coordinate is in pixel, and z is array slice index.
-//! @param source image resource which is used as the source of the filtering.
-//!               Note that it contains entire of the shadowmap atlas, not a single shadowmap.
-//! @param direction either (1,0) or (0,1).  
-//!                   If (1,0), the filtering direction is horizontal,
-//!                   and if (0,1), it is vertical.
-//! @param sourceMin the minimum (left/top most) index of the shadowmap.
-//! @param sourceMax the maximum (right/bottom most) index of the shadowmap.
-//! @param filterTable the weight table for this table.
-//!                    Since the weight table of a Gaussian filter is left-right symmetry,
-//!                    the right half is omitted in this filterTable.
-//! @param filterOffset the offset of the filtering parameter in filterTable.
-//! @param filterCount the element count of filtering parameter in filterTable.
-//!                        For example, the weight table has size 11 in the original meaning
-//!                        of Gaussian filter, filterCount == 6 by omitting the right half.
-float FilteredFloat(
-    uint3 coord,
-    Texture2DArray<float> source,
-    int2 direction,
-    int sourceMin,
-    int sourceMax,
-    Buffer<float> filterTable,
-    uint filterOffset,
-    uint filterCount)
+float BilinearFilter(const float tl, const float tr, const float bl, const float br, const float2 uv, const float texMapSize)
```

Comment:
Since the parameters names are not self explanatory, either change them to use or add short comment explaining each (even a single remark about the acronym br = bottom right..) will help.

Reply:
Will fix